vue.config.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  // devServer: {
+  //   proxy: {
+  //     "/api": {
+  //       target: "http://localhost:3008/",
+  //       changeOrigin: true,
+  //     },
+  //   },
+  // },
```

Comment:
```suggestion
```
After these and the respective changes to your api urls the vite build will no longer work correctly.
Consider either changing vue-cli config to match the vite one or simplify vite to no longer handle appended `api` to the start of urls.

Reply:
I ditched out vite. Prefer to continue with vue cli.